imcsdk/apis/server/adaptor.py

```diff
@@ -13,61 +13,121 @@
 
 
 """
-This module provides apis to setup cisco vic adaptor properties and create vnics
-and vhbas
+This module provides apis to setup cisco vic adaptor properties \
+and create vnics and vhbas
 """
 
+from imcsdk.imccoreutils import get_server_dn
+from imcsdk.imcexception import ImcOperationError
 
-def get_vic_adaptor_properties(handle, adaptor_slot):
+
+def _is_valid_arg(param, kwargs):
+    return param in kwargs and kwargs[param] is not None
```

Comment:
`param in kwargs and kwargs[param] is not None` can also be written as `return kwargs.get(param) is not None`